mail_activity_board/static/src/js/override_chatter.js

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2018 David Juaneda
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+odoo.define('mail.Chatter.activity', function(require){
+    "use strict";
+
+    var chatter = require('mail.Chatter');
+
+    chatter.include({
+
+        events: {
```

Comment:
I hope you're not overriding parent events. To protect against that, you should do:

```js
events: _.extend({}, chatter.prototype.events, {
    // your events here...
}),
```

JS is funny :yum:

Reply:
When you know what little you've learned by hitting your head against the wall, ... it's very funny, it's not just.